engine/apps/api/serializers/alert_group.py

```diff
@@ -242,3 +242,8 @@ def get_limited_alerts(self, obj: "AlertGroup"):
 
     def get_paged_users(self, obj: "AlertGroup") -> typing.List[PagedUser]:
         return obj.get_paged_users()
+
+
+class AlertGroupUpdateSerializer(AlertGroupSerializer):
+    class Meta(AlertGroupSerializer.Meta):
+        read_only_fields = [field for field in AlertGroupSerializer.Meta.fields if field not in ["grafana_incident_id"]]
```

Comment:
Curious, could we instead set `fields = ["grafana_incident_id"]` and limit the serializer/update to only work for that field?